verification/curator-service/ui/src/components/landing-page/LandingPage.tsx

```diff
@@ -82,6 +84,7 @@ interface CognitoUserExtended extends CognitoUser {
 
 interface LandingPageProps {
     setUser: (user: User | undefined) => void;
+    setRegistrationScreenOn?: () => void;
```

Comment:
You can remove `setRegistrationScreenOn` from `LandingPageProps` as it isn't used anywhere